src/nft/components/profile/list/utils.ts

```diff
@@ -109,7 +109,7 @@ const getListings = (sellAssets: WalletAsset[]): [CollectionRow[], ListingRow[]]
         )
       ) {
         const newCollectionRow = {
-          images: [asset.asset_contract.image_url, marketplace.icon],
+          images: [asset.asset_contract.image_url],
```

Comment:
nit: this could be refactored as the array is no longer needed, `image: asset.asset_contract.image_url`